doc/releases/changelog-dev.md

```diff
@@ -93,6 +93,9 @@
 
 <h4>Other Improvements</h4>
 
+* Added PyTree support for measurements in a circuit. 
+  [(#6378)](https://github.com/PennyLaneAI/pennylane/pull/6378)
+
```

Comment:
Potentially worth an example here.  I think this is a feature that most users could benefit from , but saying "Add Pytree support" isn't going to mean anything to the vast majority of users.

Reply:
Done!